### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by clavo2** [KYDronePilot/hdfm#23]
Thoughts on this error:

 DEBUG [hdfm] Application::view method called
2022-09-10T08:29:12.500Z INFO [hdfm] Artwork update
2022-09-10T08:29:12.500Z INFO [hdfm] Weather map update
2022-09-10T08:29:12.500Z INFO [hdfm] Traffic map update

**2022-09-10T08:29:12.500Z DEBUG [hdfm::weather] No weather config found in /var/folders/22/83j5qh4x56943pvr3xzwknhm0000gn/T/.tmpHxDwLg**

2022-09-10T08:29:12.501Z DEBUG [hdfm] Application::view method called
2022-09-10T08:29:12.561Z DEBUG [hdfm] Applica

Followed instructions on a new mac M1.  
It loads the GUI and populates song data, says it's receiving weather, but never generates any.  

Thanks

**Comment by ferrellsl** [comment]
Are you sure the station you're tuning to even broadcasts weather and traffic data?  Most digital stations here in the Phoenix valley do not and on those stations I get the same message until I tune to a station that does provide such data.  There are 11 digital stations in the Phoenix valley and only 4 of them are providing traffic and weather data.

 

    On Saturday, September 10, 2022 at 01:33:07 AM MST, clavo2 ***@***.***> wrote:  
 
 


Thoughts on this error:

DEBUG [hdfm] Application::view method called
2022-09-10T08:29:12.500Z INFO [hdfm] Artwork update
2022-09-10T08:29:12.500Z INFO [hdfm] Weather map update
2022-09-10T08:29:12.500Z INFO [hdfm] Traffic map update

2022-09-10T08:29:12.500Z DEBUG [hdfm::weather] No weather config found in /var/folders/22/83j5qh4x56943pvr3xzwknhm0000gn/T/.tmpHxDwLg

2022-09-10T08:29:12.501Z DEBUG [hdfm] Application::view method called
2022-09-10T08:29:12.561Z DEBUG [hdfm] Applica

Followed instructions on a new mac M1.
It loads the GUI and populates song data, says it's receiving weather, but never generates any.

Thanks

**Comment by KYDronePilot** [comment]
Yeah @clavo2, check and see if your station is listed here: https://en.m.wikipedia.org/wiki/List_of_radio_stations_owned_by_iHeartMedia. Only radio stations owned by iHeartMedia broadcast weather and traffic data.
